extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetTableLocation.java

```diff
@@ -257,7 +257,8 @@ public BasicDataIndex loadDataIndex(@NotNull final String... columns) {
                             getParquetKey().getURI(), Arrays.toString(columns)));
         }
         // Create a new index from the parquet table
-        final Table table = readDataIndexTable(getParquetKey().getURI(), indexFileMetaData, ParquetInstructions.EMPTY);
+        final Table table = readDataIndexTable(getParquetKey().getURI(), indexFileMetaData,
+                readInstructions.withTableDefinitionAndLayout(null, ParquetInstructions.ParquetFileLayout.SINGLE_FILE));
```

Comment:
The `withTableDefinitionAndLayout` might be better *inside* `readDataIndexTable`.